x-pack/plugins/secops/public/pages/overview/overview.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  // @ts-ignore
+  EuiFlexGroup,
+} from '@elastic/eui';
+import React from 'react';
+import { pure } from 'recompose';
+
+import { OverviewHost } from '../../components/page/overview/overview_host';
+import { SiemOverviewNetwork } from '../../components/page/overview/overview_network';
+import { GlobalTime } from '../../containers/global_time';
+import { PageContent, PageContentBody } from '../styles';
+
+import { Summary } from './summary';
+import { Welcome } from './welcome';
+
+const LABEL: string = 'SIEMOverview';
+
```

Comment:
We are using manageQuery in the other pages to be able to re-fetch the data if the user clicks on `Refresh`. I know that the implementation of manageQuery is not optimyze yet but at least it will give you the `Refresh` button working